packages/utilities/src/collections/index.js

```diff
@@ -1,98 +1,102 @@
 /**
- * Copyright 2017 - 2024 IBM Corporation.
- * SPDX-License-Identifier: Apache2.0
+ * @file
+ * @copyright IBM Corporation 2017–2024
+ * @license Apache-2.0
```

Comment:
This looks leftover from the previous changes that are now split out into #703 

I think it's worth keeping this commit clean